app/src/main/AndroidManifest.xml

```diff
@@ -282,6 +282,9 @@
         <meta-data
```

Comment:
Should we remove the temporary one as well if we're using the permanent?

Reply:
It looks like people include both in their manifests and it probably won't hurt? 🤷‍♀️